src/Http/Middleware/ForceTwoFactor.php

```diff
@@ -22,10 +22,10 @@ public function handle(Request $request, Closure $next): mixed
             if ($currentPanel) {
                 return redirect()->to(route('filament.' . $currentPanel->getId() .  '.pages.two-factor', [
                     'tenant' => Filament::getTenant(),
-                ]));
+                ]))->with('two_factor_redirect_message', __('Your administrator requires you to enable two-factor authentication.'));
```

Comment:
Can we make this customizable?
Example:

`->forced(message: __('Some message')`

Reply:
Yes maybe it's better to let the developer choose which message to show to the user. I'm also not really satisfied with showing the message yet. Weird enough I couldn't show the message in the blade view itself using the `session` helper..